msal/src/main/java/com/microsoft/identity/client/internal/MsalUtils.java

```diff
@@ -259,7 +260,8 @@ public static boolean hasCustomTabRedirectActivity(@NonNull final Context contex
         for (final ResolveInfo info : resolveInfoList) {
             final ActivityInfo activityInfo = info.activityInfo;
 
-            if (activityInfo.name.equals(BrowserTabActivity.class.getName())) {
+            if (activityInfo.name.equals(BrowserTabActivity.class.getName()) ||
+                    activityInfo.name.equals(CurrentTaskBrowserTabActivity.class.getName())) {
```

Comment:
I guess we need to be looking for a specific one based on library configuration? based on this code it seems the check would succeed if filter for `BrowserTabActivity` is defined but filter for `CurrentTaskBrowserTabActivity` is not defined...but what if the user specified to use `authorizartionInCurrentTast`? I guess they are destined to fail later in the code? Why not fail here?

Reply:
Good catch.  I'll fix this.

Reply:
I fixed this here and above in checkIntentFilterAddedToAppManifestForBrokerFlow....